rasa/utils/common.py

```diff
@@ -104,11 +104,15 @@ def update_socketio_log_level() -> None:
 def update_tensorflow_log_level() -> None:
     """Set the log level of Tensorflow to the log level specified in the environment
     variable 'LOG_LEVEL_LIBRARIES'."""
+
+    os.environ[
+        "TF_CPP_MIN_LOG_LEVEL"
+    ] = "2"  # Disables libvinfer, tensorRT, cuda, AVX2 and FMA warnings (CPU support)
+
     import tensorflow as tf
 
     log_level = os.environ.get(ENV_LOG_LEVEL_LIBRARIES, DEFAULT_LOG_LEVEL_LIBRARIES)
 
-    os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"  # disables AVX2 FMA warnings (CPU support)
```

Comment:
Why did we move it up? Does it need to be before the import statement? Maybe we should add a comment if position is important.